org-formation/710-tgw/vpc.yaml

```diff
@@ -73,16 +77,36 @@ Resources:
         - - !Ref VpcSubnetPrefix
           - !FindInMap [SubnetConfig, SubnetC, CIDR]
       VpcId: !Ref VPC
-#  SubnetD:
-#    Type: "AWS::EC2::Subnet"
-#    Properties:
-#      AvailabilityZone: !Select [3, !GetAZs '']
-#      CidrBlock: !Join
-#        - '.'
-#        - - !Ref VpcSubnetPrefix
-#          - !FindInMap [SubnetConfig, SubnetD, CIDR]
-#      VpcId: !Ref VPC
-#      MapPublicIpOnLaunch: true
+  SubnetD:
+    Type: "AWS::EC2::Subnet"
+    Properties:
+      AvailabilityZone: !Select [0, !GetAZs '']
```

Comment:
SubnetA is already select [0] from !GetAZs... this will not conflict?

Reply:
i don't think so because the VPN is no longer associated with private subnets (A,B, and C).